.github/workflows/docker-x86.yml

```diff
@@ -0,0 +1,74 @@
+name: Build and test x86 Docker Images
```

Comment:
Can you add a descriptive comment in the top to clarify the purpose of having this dedicated workflow file?

The point as I understand it, is to run another workflow that will finish a lot quicker, allowing us to conclude if things work or not far quicker, when we don't expect there to be any platform specific breaking change.

Reply:
Yes, you got it right. Done.